core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search-list/dot-category-field-search-list.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DotCategoryFieldSearchListComponent } from './dot-category-field-search-list.component';
+
+describe('DotCategoryFieldSearchListComponent', () => {
+    let component: DotCategoryFieldSearchListComponent;
+    let fixture: ComponentFixture<DotCategoryFieldSearchListComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
```

Comment:
@oidacra We should use spectator as a standard for testing.

Reply:
Yes, changed to spectator.